src/renderer/thread_render_context.rs

```diff
@@ -108,7 +109,13 @@ pub struct RenderId(usize);
 
 // Having this alias allows us to change the Display struct later without changing the type name
 // all throughout our code
+//
+// On Windows headless causes issues when compiled with electron, so we use a
+// hidden display instead (see build_display).
+#[cfg(not(target_os = "windows"))]
```

Comment:
@sunjay do you know if there's a nice way to tie this and `build_display` together?

I saw something about using `cfg_attr` and including modules instead but not sure if that's overkill (a windows.rs and not-windows.rs ?!)

Reply:
I think this is good as it is. We could do the module thing, but it probably isn't worth it for this little code.